### PR TITLE
Final changes to supply caps; proposal text; WETH transfer amount

### PR DESCRIPTION
**Pull request by jflatow**

### CODE REVIEW

File: deployments/mainnet/weth/migrations/1666906000_configurate_seed_reserves_and_rewards.ts [hunk 1]
```diff
@@ -66,19 +66,29 @@ export default migration('1666906000_configurate_seed_reserves_and_rewards', {
...
...this, using the entire [Comet scenario suite](https://github.com/compound-finance/comet/tree/main/scenario).\n\nAlthough the proposal sets the entire configuration in the Configurator, the initial deployment already has most of these same parameters already set. The only actual differences in parameters from the existing deployment implementation are: \n - to raise the supply caps to 80,000 wstETH and 9,000 cbETH, and\n - to set the COMP supply rewards speed to ~38.7 COMP per day, corresponding to the amount cut from cETH(v2)\n\n## Proposal\n\nThe first proposal action sets the cETH(v2) COMP reward speeds to zero in the Comptroller.\n\nThe second action sets the CometFactory for the new Comet instance in the existing Configurator.\n\nThe third action configures the Comet instance in the Configurator.\n\nThe fourth action deploys an instance of the newly configured factory and upgrades the Comet instance to use that implementation.\n\nThe fifth action configures the existing rewards contract for the newly deployed Comet instance.\n\nThe sixth action is to wrap 1,000 ETH from the Timelock, and finally transfer 1,000 WETH to the new Comet instance, in order to seed reserves.\n" // XXX
```
Comment (scott-silver) [inline deployments/mainnet/weth/migrations/1666906000_configurate_seed_reserves_and_rewards.ts:91]: link ends with a newline:

```
[deployment pull request](https://github.com/compound-finance/comet/pull/587\n)
```

Comment (jflatow) [reply deployments/mainnet/weth/migrations/1666906000_configurate_seed_reserves_and_rewards.ts:91]: Ok this proposal text is still XXX and going to be updated again (also as TODO on the branch being merged into)

File: deployments/mainnet/weth/migrations/1666906000_configurate_seed_reserves_and_rewards.ts [hunk 2]
```diff
@@ -66,19 +66,29 @@ export default migration('1666906000_configurate_seed_reserves_and_rewards', {
...
 
-      // 6. Wrap ETH as WETH and send from Timelock to Comet to seed reserves
+      // 6. Wrap some ETH as WETH
       {
         contract: WETH,
         signature: "deposit()",
         args: [],
-        value: exp(1_000, 18)
-      }, {
+        value: 358052565869157684316n, // 500e18 - current balance
```
Comment (kevincheng96) [inline deployments/mainnet/weth/migrations/1666906000_configurate_seed_reserves_and_rewards.ts:74]: Replace with `500e18 - WETH.balanceOf(timelock.address)`?

**Review by scott-silver — APPROVED** [review]

**Review by kevincheng96 — APPROVED** [review]

File: deployments/mainnet/weth/migrations/1666906000_configurate_seed_reserves_and_rewards.ts (hunk 2 above)
Comment (jflatow) [reply deployments/mainnet/weth/migrations/1666906000_configurate_seed_reserves_and_rewards.ts:74]: It's tied to the 500 number though, and the checks we do

Comment (jflatow) [reply deployments/mainnet/weth/migrations/1666906000_configurate_seed_reserves_and_rewards.ts:74]: Actually nevermind 500 - could be good? But idk I think this is actually good/fine and checks are almost done

**Pull request merged**